src/io.jl

```diff
@@ -65,7 +65,7 @@ end
 
 ##########################################################################
 
-pyio_jl(self::PyObject) = unsafe_pyjlwrap_to_objref(self["io"].o)::IO
+pyio_jl(self::PyObject) = unsafe_pyjlwrap_to_objref(PyPtr(self["io"]))::IO
```

Comment:
Maybe just add a `PyObject` method to `unsafe_pyjlwrap_to_objref`?

Reply:
In fact, it already exists.